### PR TITLE
update vendor test suite and checkin checkout times

### PR DESCRIPTION
**Pull request by sachin-raja**

### CODE REVIEW

File: src/api-reference/direct-connects/hotel-service-4/v4.endpoints.md [hunk 1]
```diff
@@ -1712,7 +1714,7 @@ Details about nightly price for a given date range.
 
 |Name|Type|Format|Description|
 |---|---|---|---|
-`cancelDeadline`|`string`|`YYYY-MM-DDThh:mm:ssZ`|**Required** Date string in date-time notation as defined by RFC 3339 with time representing local time at property and corresponding timezone. Timezone can be skipped if not available and just provide date-time in local time. |
```
Comment (kunalshah01) [inline src/api-reference/direct-connects/hotel-service-4/v4.endpoints.md:1715]: let's add multiple format here as well `YYYY-MM-DDThh:mm:ssZhh:mm` or `YYYY-MM-DDThh:mm:ss`

File: src/api-reference/direct-connects/hotel-service-4/v4.endpoints.md [hunk 2]
```diff
@@ -463,7 +463,7 @@ POST /hotels/ratedetails
       "cancelPenalties": {
         "penalties": [
           {
-            "cancelDeadline": "2017-07-21T17:32:28Z",
+            "cancelDeadline": "2017-07-21T17:32:28Z07:00",
```
Comment (kunalshah01) [inline src/api-reference/direct-connects/hotel-service-4/v4.endpoints.md:466]: let's change some of these examples to remove timezone

**Review by kunalshah01 — APPROVED** [review]

File: src/api-reference/direct-connects/hotel-service-4/v4.endpoints.md (hunk 1 above)
Comment (sachin-raja) [reply src/api-reference/direct-connects/hotel-service-4/v4.endpoints.md:1715]: updated

**Review by RoAnnC1 — APPROVED** [review]
:shipit:

**Pull request merged**